File: app/src/main/java/com/roombooking/model/Pass.java

package com.roombooking.model;

import com.google.gson.annotations.SerializedName;

public class Pass {

    @SerializedName("email")
    private String mEmail;
    @SerializedName("name")
    private String mName;
    @SerializedName("number")
    private String mNumber;

    public String getEmail() {
        return mEmail;
    }

    public void setEmail(String email) {
        mEmail = email;
    }

    public String getName() {
        return mName;
    }

    public void setName(String name) {
        mName = name;
    }

    public String getNumber() {
        return mNumber;
    }

    public void setNumber(String number) {
        mNumber = number;
    }

}

File: app/src/main/java/com/roombooking/model/Pass.kt
package com.roombooking.model

import com.google.gson.annotations.SerializedName

class Pass {

    @SerializedName("email")
    var email: String? = null
    @SerializedName("name")
    var name: String? = null
    @SerializedName("number")
    var number: String? = null

}
